x-pack/plugins/security/server/authorization/privileges/privileges.test.ts

```diff
@@ -50,32 +50,23 @@ describe('features', () => {
       all: [
         actions.login,
         actions.version,
-        actions.app.get('app-1'),
-        actions.app.get('app-2'),
-        actions.ui.get('catalogue', 'catalogue-1'),
-        actions.ui.get('catalogue', 'catalogue-2'),
-        actions.ui.get('management', 'foo', 'management-1'),
-        actions.ui.get('management', 'foo', 'management-2'),
         actions.ui.get('navLinks', 'kibana:foo'),
         actions.allHack,
       ],
-      read: [
+      minimal_all: [
         actions.login,
         actions.version,
-        actions.app.get('app-1'),
-        actions.app.get('app-2'),
-        actions.ui.get('catalogue', 'catalogue-1'),
-        actions.ui.get('catalogue', 'catalogue-2'),
-        actions.ui.get('management', 'foo', 'management-1'),
-        actions.ui.get('management', 'foo', 'management-2'),
         actions.ui.get('navLinks', 'kibana:foo'),
+        actions.allHack,
       ],
+      read: [actions.login, actions.version, actions.ui.get('navLinks', 'kibana:foo')],
+      minimal_read: [actions.login, actions.version, actions.ui.get('navLinks', 'kibana:foo')],
     });
   });
 
   test('actions defined at the privilege take precedence', () => {
```

Comment:
Do we even need this test anymore since there's no cascading?

Reply:
Nope!